tray/actions.go

```diff
@@ -75,6 +77,17 @@ func (ti *Instance) logout(persistToken bool) bool {
 	}
 }
 
+func (ti *Instance) notifyServiceExpired(url string, trustedPassURL string) {
+	resp, err := ti.client.TokenInfo(context.Background(), &pb.Empty{})
+	isTokenDataValid := resp.TrustedPassToken != "" && resp.TrustedPassOwnerId != ""
+
+	link := url
+	if err == nil && isTokenDataValid {
+		link = fmt.Sprintf(trustedPassURL, resp.TrustedPassToken, resp.TrustedPassOwnerId)
+	}
+	ti.notify(fmt.Sprintf(cli.ExpiredAccountMessage, link))
```

Comment:
Do we want to display the same message "Your account has expired. Renew your subscription now to continue enjoying the ultimate privacy and security with NordVPN." for both account expiration and dedicated IP expiration?

Reply:
I think that `NoDedicatedIPMessage = "You don’t have a dedicated IP subscription. To get a personal IP address that belongs only to you, continue in the browser: %s"` should be displayed in the case of dedicated IP expiration.

Reply:
Fixed